src/plugins/fcrypt/README.md

```diff
@@ -0,0 +1,62 @@
+- infos = Information about fcrypt plugin is in keys below
+- infos/author = Peter Nirschl <peter.nirschl@gmail.com>
+- infos/licence = BSD
+- infos/provides = sync filefilter crypto
+- infos/needs =
+- infos/recommends =
+- infos/placements = pregetstorage postgetstorage precommit
+- infos/ordering = sync
+- infos/status = experimental unfinished discouraged
+- infos/metadata =
+- infos/description = File Encryption
+
+# fcrypt Plugin #
+
+## Introduction ##
+
+This plugin enables file based encryption and decryption using GPG.
+
+This plugin encrypts backend files before the commit is executed (thus `precommit`).
+The plugin decrypts the backend files before the getstorage opens it (thus `pregetstorage`).
+After the getstorage plugin has read the backend file, the plugin decrypts the backend file again (thus `postgetstorage`).
+
+## Security Considerations ##
+
+During decryption the plugin temporarily writes the decrypted plain text to the same directory as the original (encrypted) file.
+This is a vulnerability as an attacker might have access to the plain text for a short period of time (the time between pregetstorage and postgetstorage calls).
```

Comment:
maybe you can add here more about that the file will be shredded, except when the process crashes.